applications/accounts-api/backend/accounts_api/openapi/openapi.yaml

```diff
@@ -46,6 +46,31 @@ paths:
       - infrastructure
       x-openapi-router-controller: accounts_api.controllers.infrastructure_controller
   /users:
+    get:
```

Comment:
need to add the bearerAuth security, otherwise the api is public

Reply:
hrm, but the `/user/<userid>` end point is also open at the moment?

https://api.accounts.v2dev.opensourcebrain.org/api/ui/#/users/get_user

Reply:
It's open with the trick of removing the email if you are not logged in. It must be open as it's necessary to show the user page to not logged users. It may be fine to leave also this one open but I don't like much the user listing to be open honestly

Reply:
We could probably use a similar trick in this end point then? At the moment, I just remove the e-mail address. But we could make it better to return a limited set of fields for not logged in users, and more for logged in users?

The use case for allowing some information being accessible without logging in is for the user-facing user page that we want to add to the portal later. We want people to be able to come to v2 to see all the labs/users that are using it---and for this I expect we want them to be able to do it without creating an account, right?

Reply:
Agree a similar trick would be ok, I'n not sure about the regulations though as listing users subscribed to a platform is not something you see around usually

Reply:
Yeh, that's true. I think even for GitHub, one can query contributors for a specific repository, but not get a full list of platform users. The user facing user page can be more of a "timeline" perhaps which shows activity on the platform rather than a list of all users or something. We could also just provide summary information to not logged in users: "X workspaces, Y repositories, Z users", and then some more information after they login etc.

Let me make this end point limited for now, and we can see if/how it needs tweaking when we design the user-facing user page.

Reply:
Should be done now :+1: